mac.js

```diff
@@ -12,7 +12,7 @@ class MacApp {
   constructor (opts, stagingPath) {
     this.opts = opts
     this.stagingPath = stagingPath
-    this.appName = opts.name
+    this.appName = common.sanitizeAppName(opts.name)
```

Comment:
OK, now that I've committed https://github.com/electron-userland/electron-packager/pull/455/commits/7cdec7df85e207e476af497c671fd10bef7e01a5 I feel a bit better about the completeness of this PR, and removed the `[WIP]`. Reviews are much appreciated.